universal-login-react/src/ui/TopUp/TopUp.tsx

```diff
@@ -40,6 +47,7 @@ export const TopUp = ({sdk, onGasParametersChanged, contractAddress, startModal,
           <ChooseTopUpMethod
             contractAddress={contractAddress}
             onPayClick={onPayClick}
+            ipGeolocationApiConfig={ipGeolocationApiConfig}
```

Comment:
Why not skip all the passing of values and just do:
```ts
ipGeolocationApiConfig={sdk.realyerConfig.ipGeolocationApiConfig}
```

Same for `onRampConfig`?

Reply:
done in https://github.com/UniversalLogin/UniversalLoginSDK/pull/1201